### PR TITLE
Feature/preset and helpers

### PR DESCRIPTION
**Pull request by jakublamprecht**
Updated structure and created strudel-jest preset. Also added helpers for writing unit tests for Strudel components and updated README

### CODE REVIEW

**Comment by mateuszluczak** [comment]
General comment - I believe the convention that jest uses is
`jest-preset-*`
So we should rename the repo/package to follow that convention

**Comment by jakublamprecht** [comment]
> General comment - I believe the convention that jest uses is
> `jest-preset-*`
> So we should rename the repo/package to follow that convention

This isn't well documented by Jest to be honest, but it seems like the community seems to be creating presets that way. React-native is an exception. Just wondering about importing the helpers - "import { createComponentWrapper } from 'jest-preset-strudel'" - are we ok with that?

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
 # strudel-jest
+
+This package provides jest config preset and helpers for writing unit tests for Strudel components.
+
+## Installation
+
+To use strudel-jest you must have the following installed and configured:
+
+* jest
```
Comment (mateuszluczak) [inline README.md]: This information I believe is redundant, should pretty self explainatory that strudel jest requires jest :)

Comment (jakublamprecht) [reply README.md]: I agree, removed 😅

**Pull request merged**